sklearn/linear_model/coordinate_descent.py

```diff
@@ -331,6 +332,10 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
     positive : bool, default False
         If set to True, forces coefficients to be positive.
 
+    check_input : bool, default True
+        Skip input validation checks, including the Gram matrix when provided
+        assuming there are handled by the caller when check_input=False.
+
```

Comment:
Yes I think so: `check_input` flags are already present in several other functions / methods in the scikit-learn code base.